From 5c2d7c4dc8b8618f6586e220f4af6e4baae6dac2 Mon Sep 17 00:00:00 2001
From: Reinhard Tartler <siretart@tauware.de>
Date: Tue, 12 Jul 2011 18:09:55 +0200
Subject: [PATCH 50/57] docs: Don't recommend adding --enable-memalign-hack
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It is enabled automatically when required nowadays.

Signed-off-by: Martin Storsjö <martin@martin.st>
(cherry picked from commit 9d36139231bfcf155a1b94f61a420768f1771174)
---
 doc/general.texi |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/general.texi b/doc/general.texi
index dbc7faf..93f563e 100644
--- a/doc/general.texi
+++ b/doc/general.texi
@@ -949,7 +949,7 @@ MSVC++-compatible import libraries.
 @item Build Libav with
 
 @example
-./configure --enable-shared --enable-memalign-hack
+./configure --enable-shared
 make
 make install
 @end example
@@ -1076,7 +1076,7 @@ For a static build run
 
 and for a build with shared libraries
 @example
-./configure --target-os=mingw32 --enable-memalign-hack --enable-shared --disable-static --extra-cflags=-mno-cygwin --extra-libs=-mno-cygwin
+./configure --target-os=mingw32 --enable-shared --disable-static --extra-cflags=-mno-cygwin --extra-libs=-mno-cygwin
 @end example
 
 @bye
-- 
1.7.4.1

